fem/datacollection.hpp

```diff
@@ -32,12 +32,23 @@ class DataCollection
    typedef std::map<std::string, GridFunction*> FieldMapType;
    typedef std::map<std::string, QuadratureFunction*> QFieldMapType;
 
+   /// Format constants to be used with SetFormat().
+   enum Format
+   {
+      FMT_SERIAL = 0, /**<
```

Comment:
Can these be named simply `SERIAL` and `PARALLEL`, as in:
```
visit_dc.SetFormat(DataCollection::SERIAL);
```

If we have to express format, I prefer `DataCollection::SERIAL_FORMAT`.

Reply:
I prefer to have 'format' (in some form) in the names. I'll change it to `SERIAL_FORMAT` and `PARALLEL_FORMAT`.